src/main/java/net/dancier/dancer/chat/ChatService.java

```diff
@@ -0,0 +1,76 @@
+package net.dancier.dancer.chat;
+
+import lombok.RequiredArgsConstructor;
+import net.dancier.dancer.chat.client.ChatServiceClient;
+import net.dancier.dancer.chat.client.RemoteCreateMessageDto;
+import net.dancier.dancer.chat.dto.*;
+import net.dancier.dancer.core.DancerService;
+import net.dancier.dancer.core.exception.BusinessException;
+import net.dancier.dancer.core.model.Dancer;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class ChatService {
+
+    private final DancerService dancerService;
+
+    private final ChatServiceClient chatServiceClient;
+
+    public ChatsDto getChatsByUserId(UUID userId) {
+        Dancer currentDancer = dancerService.loadByUserId(userId);
+        return chatServiceClient.getChats(currentDancer.getId());
+    }
+
+    public ChatDto createChat(UUID userId, CreateChatDto createChatDto) {
+        Dancer currentDancer = dancerService.loadByUserId(userId);
```

Comment:
As we talked, please switch to let the ChatService only use the functional id (is this the right name for it) dancier_id as parameters here.
The business layer/domain should only have dependencies to it-self. I would consider the Account domain as a kind of technical domain.